src/focus.service.ts

```diff
@@ -458,7 +458,7 @@ export class FocusService {
     const rect = el.getBoundingClientRect();
     const { width, height, top, left } = rect;
 
-    for (let parent = el.parentElement; parent !== container.parentElement; parent = parent.parentElement) {
+    for (let parent = el.parentElement; parent !== container.parentElement && parent; parent = parent.parentElement) {
```

Comment:
This is coming somewhat annoying to read, perhaps we should create a iterator utility for this.